avae/config.py

```diff
@@ -61,6 +67,9 @@
     "vis_pos": VIS_POS,
     "vis_acc": VIS_ACC,
     "vis_his": VIS_HIS,
+    "vis_sim": VIS_SIM,
     "vis_all": False,
     "model": "a",
+    "gaussian_blur": False,
+    "normalise": False,
```

Comment:
Add these two params to the default config too

Reply:
Im not sure what you mean here. The one you have commented on, are the default values, right ?